src/main/groovy/io/saagie/plugin/dataops/utils/SaagieUtils.groovy

```diff
@@ -1249,6 +1249,25 @@ class SaagieUtils {
 		outputStream.close()
 		inputStream.close()
 	}
+	// This function is helpful to get for example size of a file without downloading it
+	
+	int getOnlyHeaderInformationFromApi(String url, OkHttpClient client, version) {
```

Comment:
If it's only for file, maybe rename it to getFileInformationsFromApi, I think it's enougth